examples/src/components/utils/ExampleWithExport.jsx

```diff
@@ -22,7 +21,7 @@ export class ExampleWithExport extends React.Component {
         this.exportToXLSX = this.exportToXLSX.bind(this);
         this.exportWithCustomName = this.exportWithCustomName.bind(this);
         this.exportWithDialog = this.exportWithDialog.bind(this);
-        this.exportWithMergeHeaders = this.exportWithMergeHeaders.bind(this);
+        this.exportDialogSubmit = this.exportDialogSubmit.bind(this);
         this.doExport = this.doExport.bind(this);
         this.getExportDialog = this.getExportDialog.bind(this);
         this.exportDialogCancel = this.exportDialogCancel.bind(this);
```

Comment:
please consider transform these functions to arrow functions so that we can stop using `.bind(this)`
or at least just the new added one.

Reply:
updated